doc/source/examples/analysis/custom_parallel_analysis.ipynb

```diff
@@ -0,0 +1,986 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "result = np.asarray(result).T\n",
+    "\n",
+    "labels = ['all', 'x-axis', 'y-axis', 'z-axis']\n",
+    "for col, label in zip(result, labels):\n",
+    "    plt.plot(col, label=label)\n",
+    "plt.legend()\n",
+    "plt.ylabel('Radius of gyration (Å)')\n",
+    "plt.xlabel('Frame');"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can also use the old `radgyr` function because `dask` is more flexible on the input args.\n",
+    "\n",
+    "Note: `protein` will change its state during the trajectory iteration, so the processes are always aware which timestep the trajectory is in and change the `protein` (e.g. positions) accordingly."
```

Comment:
Can you rearrange this to say `coordinates`, as it's ambiguous how protein is changed?